lib/bolt/plugin/prompt.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'concurrent'
```

Comment:
This is causing a regression in our integration test for lazily loading gems (https://github.com/puppetlabs/bolt/blob/71d37234dc8c4635c147c5fdd6c650649cf7dec7/spec/integration/cli_spec.rb#L30). Eventually will we need a solution for loading gem libraries specific to the plugin being used? Also it looks like I may be able to just require `concurrent/delay`.

Reply:
Long term I think we will lazily require/load this file so plain requires should be fine.

Reply:
cool. that makes life easy. For now i'm removing concurrent-ruby from https://github.com/puppetlabs/bolt/pull/994/commits/b9ce42bb0c8f2e41fcd2156420b94b9eb79700ee#diff-fb52e06d3eb62217257cc824e4323d80L30